index.js

```diff
@@ -0,0 +1,43 @@
+import { DateTime } from './modules/luxon.js';
+import * as DisplaySections from './modules/displaySections.js';
+import Book from './modules/book.js';
+import Books from './modules/books.js';
+
+const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
```

Comment:
- [ ] [OPTIONAL] Great job, :+1: but I believe adding some callback function to this line will make the time and date more recent, and correct without having to refresh the page in order to get the correcting time. This will improve user experience if the time is in real-time. You can consider using a `setInterval()` method for this. Kindly check [this link](https://www.w3schools.com/jsref/met_win_setinterval.asp) as a guide.

![image](https://user-images.githubusercontent.com/99715640/227926985-95d137e2-ba9f-49d3-b45d-f4dfb30e1c86.png)

Reply:
Great. Thanks. I've implemented this too.
Thanks again